scripts/check_determinism.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Run this script from the repo root with e.g.:
+#
+#   ./scripts/check_determinism.sh rutland
+#
+# passing the region to be checked as the first positional arg.
+#
+# If files differ, the protobuf are converted to sorted JSON to get diff.
+
+# Use given region
```

Comment:
Should we `set -e` at the top and fail-fast for any bad command? First time I ran this, I failed to provide an argument, and the script output was messy

Reply:
Yes good idea! Added here (173e521d1e8359627dc2cabd12fe53e4700d9760).